packages/synapse-react-client/src/components/SynapseTable/datasets/DatasetItemsEditor.test.tsx

```diff
@@ -289,14 +283,14 @@ describe('Dataset Items Editor tests', () => {
   beforeEach(() => {
     vi.clearAllMocks()
     server.resetHandlers()
+    server.use(getEntityHandler)
   })
   afterAll(() => server.close())
 
   it('Displays call to action when there are no items', async () => {
     const { ADD_ITEMS, NO_ITEMS_IN_THIS_DATASET } = datasetCopy
 
-    const getEmptyDatasetHandler = getDatasetHandlerWithItems('dataset', [])
-    server.use(getEmptyDatasetHandler)
+    configureGetEntityHandler('dataset', [])
```

Comment:
Simplified the setup in each test. Just call this, no need to pass a handler to the mock `server`